### PR TITLE
DOCS-1375: Update I2C config attributes and screenshot for adx1345 accelerometer

### PR DESCRIPTION
**Pull request by sguequierre**
Another round of @penguinland updates

### CODE REVIEW

File: docs/components/movement-sensor/adxl345.md [hunk 1]
```diff
@@ -129,7 +128,7 @@ Then remove and fill in the attributes as applicable to your movement sensor, ac
       "namespace": "rdk",
       "attributes": {
         "board": "local",
-        "i2c_bus": "default_i2c_bus",
+        "i2c_bus": "1",
```
Comment (penguinland) [inline docs/components/movement-sensor/adxl345.md:131]: This says "1", but line 60 gave the example "2" and the new screenshot uses "2". Should they be different to show that you can have different values, or the same to be consistent? Maybe it's right the way you've written it; I'm just looking for weird stuff.

File: docs/components/movement-sensor/adxl345.md [hunk 2]
```diff
@@ -143,13 +142,13 @@ Then remove and fill in the attributes as applicable to your movement sensor, ac
...
-| `i2c_bus`             | string | **Required** | The `name` of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on the [board](/components/board/) wired to this device. |
-| `use_alternate_i2c_address` | bool   | Optional     | Depends on whether you wire SDO low (leaving the default address of 0x53) or high (making the address 0x1D). If high, set true. If low, set false or omit the attribute. <br> Default: `false` |
-| `tap`                 | object | Optional     | Holds the configuration values necessary to use the tap detection interrupt on the ADXL345. See [Tap attributes](#tap-attributes). |
-| `free_fall`           | object | Optional     | Holds the configuration values necessary to use the free-fall detection interrupt on the ADXL345. See [Freefall attributes](#freefall-attributes). |
+| Name | Type   | Inclusion    | Description |
+| ---- | ------ | ------------ | ----------- |
+| `i2c_bus` | string | **Required** | The `name` of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on the [board](/components/board/) wired to this device. |
```
Comment (penguinland) [inline docs/components/movement-sensor/adxl345.md:147]: The point of the RDK change is that you don't configure the I2C bus on the board component any more, and this component is no longer dependent on the board at all. I like the phrasing in https://docs.viam.com/components/board/pca9685/: "The index of the I2C bus on the board your device is connected to. Often a number. Example: `"1"`"

File: docs/components/movement-sensor/adxl345.md [hunk 3]
```diff
@@ -129,7 +128,7 @@ Then remove and fill in the attributes as applicable to your movement sensor, ac
       "namespace": "rdk",
       "attributes": {
         "board": "local",
```
Comment (penguinland) [inline docs/components/movement-sensor/adxl345.md:130]: We can remove this line, and frankly we can remove the entire board component above (lines 110-123). This example doesn't use any interrupt pins, so we no longer depend on the board at all.

File: docs/components/movement-sensor/adxl345.md [hunk 4]
```diff
@@ -143,13 +142,13 @@ Then remove and fill in the attributes as applicable to your movement sensor, ac
...
-| `tap`                 | object | Optional     | Holds the configuration values necessary to use the tap detection interrupt on the ADXL345. See [Tap attributes](#tap-attributes). |
-| `free_fall`           | object | Optional     | Holds the configuration values necessary to use the free-fall detection interrupt on the ADXL345. See [Freefall attributes](#freefall-attributes). |
+| Name | Type   | Inclusion    | Description |
+| ---- | ------ | ------------ | ----------- |
+| `i2c_bus` | string | **Required** | The `name` of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on the [board](/components/board/) wired to this device. |
+| `board` | string | Optional | The `name` of the [board](/components/board/) to which the device is wired. Only needed if you've configured any [interrupt](/components/board/#digital_interrupts) functionality. |
+| `use_alternate_i2c_address` | bool | Optional | Depends on whether you wire SDO low (leaving the default address of 0x53) or high (making the address 0x1D). If high, set true. If low, set false or omit the attribute. <br> Default: `false` |
```
Comment (penguinland) [inline docs/components/movement-sensor/adxl345.md:149]: I'm open to disagreement, but I'd swap lines 148 and 149. That way, the board is right next to the tap and free_fall sections that interact with it, and the fields are roughly ordered by popularity (use_alternative_i2c_address is probably used more often than board will be after this change is out).

File: docs/components/movement-sensor/adxl345.md [hunk 5]
```diff
@@ -32,7 +32,7 @@ Then remove and fill in the attributes as applicable to your movement sensor, ac
 ```json {class="line-numbers linkable-line-numbers"}
 {
   "board": "<your-board-name>",
```
Comment (penguinland) [inline docs/components/movement-sensor/adxl345.md:34]: I'd order these the same as the attribute descriptions below. If you like my other suggestion, move this line down below "use_alternate_i2c_address" and above "tap". That way, the required attributes (i.e., only "i2c_bus" any more) are at the top.

**Review by penguinland — COMMENTED** [review]
Needs some tweaks, but they're all straightforward

File: docs/components/movement-sensor/adxl345.md (hunk 2 above)
Comment (sguequierre) [reply docs/components/movement-sensor/adxl345.md:147]: Oh yes, thank you for catching this! Updating

Comment (sguequierre) [inline docs/components/movement-sensor/adxl345.md:147]:
```suggestion
| `i2c_bus` | string | **Required** | The index of the I2C bus on the board your device is connected to. Often a number. <br> Example: "1"  |
```

File: docs/components/movement-sensor/adxl345.md [hunk 6]
```diff
@@ -143,13 +142,13 @@ Then remove and fill in the attributes as applicable to your movement sensor, ac
...
-| `use_alternate_i2c_address` | bool   | Optional     | Depends on whether you wire SDO low (leaving the default address of 0x53) or high (making the address 0x1D). If high, set true. If low, set false or omit the attribute. <br> Default: `false` |
-| `tap`                 | object | Optional     | Holds the configuration values necessary to use the tap detection interrupt on the ADXL345. See [Tap attributes](#tap-attributes). |
-| `free_fall`           | object | Optional     | Holds the configuration values necessary to use the free-fall detection interrupt on the ADXL345. See [Freefall attributes](#freefall-attributes). |
+| Name | Type   | Inclusion    | Description |
+| ---- | ------ | ------------ | ----------- |
+| `i2c_bus` | string | **Required** | The `name` of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on the [board](/components/board/) wired to this device. |
+| `board` | string | Optional | The `name` of the [board](/components/board/) to which the device is wired. Only needed if you've configured any [interrupt](/components/board/#digital_interrupts) functionality. |
```
Comment (sguequierre) [inline docs/components/movement-sensor/adxl345.md:148]:
```suggestion
| `use_alternate_i2c_address` | bool | Optional | Depends on whether you wire SDO low (leaving the default address of 0x53) or high (making the address 0x1D). If high, set true. If low, set false or omit the attribute. <br> Default: `false` |
| `board` | string | Optional | The `name` of the [board](/components/board/) to which the device is wired. Only needed if you've configured any [interrupt](/components/board/#digital_interrupts) functionality. |
```

File: docs/components/movement-sensor/adxl345.md (hunk 4 above)
Comment (sguequierre) [inline docs/components/movement-sensor/adxl345.md:149]:
```suggestion
```

File: docs/components/movement-sensor/adxl345.md (hunk 1 above)
Comment (sguequierre) [inline docs/components/movement-sensor/adxl345.md:131]:
```suggestion
        "i2c_bus": "2",
```

Comment (sguequierre) [reply docs/components/movement-sensor/adxl345.md:131]: I'm good with changing it to 1, just didn't catch that difference!

File: docs/components/movement-sensor/adxl345.md (hunk 4 above)
Comment (sguequierre) [reply docs/components/movement-sensor/adxl345.md:148]: Done!

File: docs/components/movement-sensor/adxl345.md (hunk 5 above)
Comment (sguequierre) [reply docs/components/movement-sensor/adxl345.md:34]: Done

File: docs/components/movement-sensor/adxl345.md (hunk 3 above)
Comment (sguequierre) [reply docs/components/movement-sensor/adxl345.md:130]: Sounds good!

**Review by penguinland — APPROVED** [review]
LGTM!

**Review by npentrel — APPROVED** [review]

**Pull request merged**